src/crafting/CraftingManagerFromDataHelper.php

```diff
@@ -43,13 +44,18 @@ public static function make(string $filePath) : CraftingManager{
 		$itemDeserializerFunc = \Closure::fromCallable([Item::class, 'jsonDeserialize']);
 
 		foreach($recipes["shapeless"] as $recipe){
-			if($recipe["block"] !== "crafting_table"){ //TODO: filter others out for now to avoid breaking economics
+			$blockType = match ($recipe["block"]){
```

Comment:
This leaks protocol info into the API. An enum should be added for this similar to FurnaceType.